Multisig/Cross-layer/FirebaseRemoteConfig.swift

```diff
@@ -17,13 +17,16 @@ class FirebaseRemoteConfig {
         case deprecated
         case safeClaimEnabled
         case crashDebugEnabled
+        //FIXME Remove feature flag and flag handling after release
+        case intercomMigration
```

Comment:
we have released, should we remove the flag now?

Reply:
No, next pr

Reply:
is there a task for this?

Reply:
No, I wanted to do it as part of increasing the version, if you want make a ticket for it

Reply:
yes, please make a ticket. We also need to remove this value from firebase remote config.

Reply:
Will make it later